app/src/main/java/com/onboarding/parkingsystemjava/mvp/model/ReserveModel.java

```diff
@@ -23,7 +25,7 @@ public ReserveModel(ParkingDatabase database) {
 
     @Override
     public void addReservation(Reservation reservation) {
-        database.addReservations(reservation);
+        database.addReservation(reservation.getParkingLot(), reservation);
```

Comment:
why this change?
lwas better before, the lotNumb is inside de reservation. use only 1 param

Reply:
database is now a HashMap<int,Reservation>, so the parkingLot is the key, it still in the reservation, previously database was an ArrayList<Reservation>

Reply:
done